### ISSUE TITLE
Can not reach the accuracy on paper.

### ISSUE THREAD

**Issue by xiaobaoxia** [KAIST-VICLab/SkateFormer#13]
Hi there,

When I train the model from scratch, I can only achieve 93.45 on CS NTU-60, whereas the paper reports 93.6, which is a decline of 0.15.

The only change I made is setting the batch size to 32 and using gradient accumulation of 4 to emulate a batch size of 128. Could this affect the performance?

Thanks for your outstanding work!
